packages/wdio-config/tests/shim-sync.test.js

```diff
@@ -0,0 +1,52 @@
+import { executeHooksWithArgs, runFnInFiberContext, wrapCommand, hasWdioSyncSupport, executeSync, executeAsync, runSync } from '../src/shim'
+
+jest.mock('@wdio/sync', () => ({
```

Comment:
I am not sure about the value of testing module exports. If an export is missing I am sure other unit tests will fail as well.

Reply:
I don't mind to keep it as is

Reply:
Good point, I'll check

Reply:
@christian-bromann I'd insist to keep the test.
If I comment this line https://github.com/webdriverio/webdriverio/pull/4354/files#diff-ab58d4080942109b9f399dc7e8bf88feR70

all the unit tests are passing. Even though smoke tests are failing it would take an additional time to debug why failure like `Expected [object Promise] to be 'Mock Page Title'.` has happened.